### DISCUSSION TITLE
handbook/0.1/posts/20230118_cobertura-temporal_frequencia-atualizacao/

### DISCUSSION THREAD

**Discussion by giscus** [transparencia-mg/handbook#40] [Announcements]
# handbook/0.1/posts/20230118_cobertura-temporal_frequencia-atualizacao/

https://transparencia-mg.github.io/handbook/0.1/posts/20230118_cobertura-temporal_frequencia-atualizacao/

**Comment by gabrielbdornas** [comment]
@andrelamor, o link para o [Portal Federal de Dados Abertos](https://dados.gov.br/pagina/manuais-e-orientacoes) está quebrado.

**Comment by gabrielbdornas** [comment]
@vilelafm, teste!
